addon/components/kyc-form.js

```diff
@@ -4,26 +4,28 @@ import { task } from 'ember-concurrency';
 import layout from '../templates/components/kyc-form';
 import { Promise } from "rsvp";
 import Ember from 'ember';
-import { set } from '@ember/object';
 
 const { testing } = Ember;
 
 const fields = {
   bfn: {
     name: "First Name",
     hint: "User's first name",
-    required: true
+    required: true,
+    disabled: true
```

Comment:
based on slack discussion we probably dont want to disable the first name field, right?